CONTRIBUTING.md

```diff
@@ -9,6 +9,37 @@ Be sure to you have the right to do that (if you are a professional, ask your co
 
 If you include code from another project, please mention it in the Pull Request description and credit the original author.
 
+## Start a Demo API and Contribute to the Caddy Plugin
+
+Clone the project:
+
+    $ git clone https://github.com/dunglas/vulcain
+    $ cd vulcain
+
+Install the dependencies:
+
+    $ go get
+
+Run the server with [`xcaddy`](https://github.com/caddyserver/xcaddy):
+
+    $ cd caddy
+    $ CADDY_VERSION="v2.2.0-rc.3" PATH=".:$PATH" xcaddy run
```

Comment:
I think you can update this now. You can likely remove the version env from the instructions here altogether so you need to need update them every time Caddy updates 😅

Reply:
I'll check. This wasn't compiling because of a broken dependency that has been upgraded in the beta, but it was a temporary fix anyway.